html/pfappserver/lib/pfappserver/Form/Config/Roles.pm

```diff
@@ -136,6 +153,23 @@ sub validate {
 
     }
 
+    my $acls = $self->field('acls')->value;
```

Comment:
From what I can tell this will not validate that an ACL has been added to a role but some configured switches do not support dACLs (or downloadable ACL if necessary)

Looking at fullfuling this requirement from our documented list:
> Make sure the size of the ACL is compatible with all the configured equipment
> ---- If some equipment is incompatible with the ACL, return it in the warnings

Reply:
The check is done on line 167

Reply:
According to what I see in Switch.pm, if the switch doesn't support dynamic or downloadable ACLs, no warnings are returned.

That means that if you configure an ACL in a role and some equipment doesn't support dACLs at all, then there will not be any warning